planning_doc.Rmd

```diff
@@ -96,7 +117,7 @@ knitr::opts_chunk$set(echo = TRUE)
 
 ### YYYY-MM-DD 
 
-### Roll Call 
+### Roll Call for this meeting
 
 #### Discussion Points & Notes 
```

Comment:
@FenneRiemslagh can you also change 'Assign note taker' to 'Assign notetaker for this meeting'?